packages/react-cards/src/next/Card.types.ts

```diff
@@ -38,9 +38,8 @@ export type CardProps = ComponentProps &
     /** A card can have inverted background styles. */
     inverted?: boolean;
 
-    /** A card can have quiet styles. */
-    // TODO: Is this the correct name? Appears as ghost in design spec.
-    quiet?: boolean;
+    /** A card can have ghost styles. */
+    ghost?: boolean;
```

Comment:
Let's create a separate PR for this change as it affects a different package

Reply:
https://github.com/microsoft/fluentui/pull/16632